### PR TITLE
cleanup duplicate lines and license headers

### PR DESCRIPTION
**Pull request by soc1c**
- add cleanup scripts c2808d5
- remove duplicate lines a2d2591
- update license headers 5a67622
- improve author information 7587648

### CODE REVIEW

File: availability-store/src/lib.rs [hunk 1]
```diff
@@ -1,18 +1,18 @@
-// Copyright 2018 Parity Technologies (UK) Ltd.
-// This file is part of Polkadot.
+// Copyright 2017-2019 Parity Technologies (UK) Ltd.
```
Comment (Kianenigma) [inline availability-store/src/lib.rs]: Looks like this faile was created in 2018. Shouldn't be:
```suggestion
// Copyright 2018-2019 Parity Technologies (UK) Ltd.
```

?

File: erasure-coding/src/lib.rs [hunk 2]
```diff
@@ -1,18 +1,18 @@
-// Copyright 2018 Parity Technologies (UK) Ltd.
-// This file is part of Polkadot.
+// Copyright 2017-2019 Parity Technologies (UK) Ltd.
```
Comment (Kianenigma) [inline erasure-coding/src/lib.rs]: likewise

File: erasure-coding/src/wrapped_shard.rs [hunk 3]
```diff
@@ -1,18 +1,18 @@
-// Copyright 2019 Parity Technologies (UK) Ltd.
-// This file is part of Polkadot.
+// Copyright 2017-2019 Parity Technologies (UK) Ltd.
```
Comment (Kianenigma) [inline erasure-coding/src/wrapped_shard.rs]: reverse-likewise

File: network/src/collator_pool.rs [hunk 4]
```diff
@@ -1,18 +1,18 @@
-// Copyright 2018 Parity Technologies (UK) Ltd.
-// This file is part of Polkadot.
+// Copyright 2017-2019 Parity Technologies (UK) Ltd.
```
Comment (Kianenigma) [inline network/src/collator_pool.rs]: likewise

File: availability-store/src/lib.rs (hunk 1 above)
Comment (soc1c) [reply availability-store/src/lib.rs]: It's a script that streamlines the copyright headers. The benefit is you can ensure all files are prepended with a correct header. Downside is the actual year when the file was created is no longer part of the header. Would you insist on leaving the year untouched?

Comment (bkchr) [reply availability-store/src/lib.rs]: We have a script that updates every year from `x` to `x - current_year()` or `x - y` to `x - current_year`

Comment (Kianenigma) [reply availability-store/src/lib.rs]: No, I don't insist. But I recommend checking with the rest of the team.

at least in substrate, I am pretty sure that we maintain the creation date of a file persistently (or at least we try until the codebase is too large to maintain like that).

And I assume the script can actually maintain the creation date right? Just have to scrape the year and check if it needs a bump or not. If it has the format `X-Y` update `Y` else, append a `-Y`.

File: src/main.rs [hunk 5]
```diff
@@ -53,12 +53,12 @@ impl cli::Worker for Worker {
...
 fn main() -> Result<(), cli::error::Error> {
 	let version = VersionInfo {
-		name: "Parity Polkadot",
+		name: "Parity Polkadot Client.",
 		commit: env!("VERGEN_SHA_SHORT"),
 		version: env!("CARGO_PKG_VERSION"),
 		executable_name: "polkadot",
-		author: "Parity Team <admin@parity.io>",
-		description: "Polkadot Relay-chain Client Node",
+		author: "Wood/Habermeier/Paronyan/Pepyakin/Köcher/Drwięga et al.",
```
Comment (andresilva) [inline src/main.rs]: Thanks! :+1:

Comment (bkchr) [reply src/main.rs]: I also don't see why we remove parity team as author?

**Pull request closed without merging**